nss-day-cohort-38/bangazon-workforce-management-aquarii-star-system
# Training Conditional Delete

## Description

- The user is able to delete training_programs from the details page, but only those whose end_date is in the past (less than the value of today's date). 
- Jack & Alyssa provided an update to my EmployeeTrainingProgram Model

## How do I access this branch?

Getting the New Branch:
1. `git checkout master`
1. `git fetch --all`
1. `git checkout kp-training-delete`
1. `git pull origin kp-training-delete` (in case you've fetched since last I pushed up)
1. Make sure your server is running: `python manage.py runserver`

## Motivation and Context


#12 

## How Has This Been Tested?



Deleting a Past Event:
1. Navigate to: http://127.0.0.1:8000/training_program/form
1. Create an event with an end date **in the past**
1. Click on that event's hyperlinked title on the list page
1. A Delete button should display on the details page.
1. Click Delete
1. Confirm that it was deleted. 

Being Unable to Delete a Future Event:
1. Click "Add Training" to add another training
1. Create an event with an end date **in the future**
1. Click on that event's hyperlinked title on the list page
1. There should be no Delete button (a user cannot delete an event in the future)

## Screenshots (if appropriate):

## Types of changes


## Checklist:
